service/cmdcache/cmdcache.go

```diff
@@ -1,4 +1,4 @@
-package replica
+package cmdcache
```

Comment:
I don't think we need a separate package for this. Logically, I wonder if it would make sense to move this `Cache` to the `clientpb` or `clientsrv` package?

Right now, I feel we have gone too far and added too many packages.

Reply:
Another name for this concept that's become common these days is a mempool. Maybe that's what we should call the `clientpb` package?